defaults/serverConfig/server.json

```diff
@@ -32,15 +32,16 @@
   },
 
 // section used to define a connection to a server that can handle native or high privilege commands that the app server cannot  
-//  "agent": {
+ "agent": {
```

Comment:
dont add these changes... agent is commented out by default because the default is for developers that might not even have a mainframe yet.